### PR TITLE
[payment-driver] use payment.db

### PR DESCRIPTION
**Pull request by tworec**
Payment Driver service to use `payment.db`

### CODE REVIEW

File: core/serv/src/main.rs [hunk 1]
```diff
@@ -149,6 +149,7 @@ impl ServiceContext {
             Self::make_entry::<MarketService>(path, "market")?,
             Self::make_entry::<ActivityService>(path, "activity")?,
             Self::make_entry::<PaymentService>(path, "payment")?,
+            Self::make_entry::<PaymentDriverService>(path, "payment")?,
```
Comment (bidzyyys) [inline core/serv/src/main.rs:152]: `payment` twice? Why not `payment-driver`? Or even `gnt-driver`? Each db is payment platform specific.

Comment (Wiezzel) [reply core/serv/src/main.rs:152]: If the idea is that each service should have its own database, then this should be changed.

Comment (tworec) [reply core/serv/src/main.rs:152]: pls change this name to whatever you want. I'm just after splitting it from main `yagna.db`

Comment (Wiezzel) [inline core/serv/src/main.rs:152]:
```suggestion
            Self::make_entry::<PaymentDriverService>(path, "gnt-driver")?,
```

**Pull request merged**